src/options/options.css

```diff
@@ -1,4 +1,4 @@
-body {
+/* body {
   background-color: #f5f5f5;
   font-family: 'Roboto';
-}
+} */
```

Comment:
why do you comment this out? If we don't need this file, should we better delete it?

Reply:
I don't know what it is, so i just commened everything so we could use it as reference later